packages/xs-vat-worker/test/test-boot-lockdown.js

```diff
@@ -5,8 +5,13 @@ import * as fs from 'fs';
 import * as path from 'path';
 import { xsnap } from '@agoric/xsnap';
 
-const dist = async name =>
-  fs.promises.readFile(path.join(__filename, '..', '..', 'dist', name));
+const importModuleUrl = `file://${__filename}`;
+
+const asset = async (...segments) =>
+  fs.promises.readFile(
+    path.join(importModuleUrl.replace('file:/', ''), '..', ...segments),
```

Comment:
Can also use URL math to eliminate the dep on Node.js-specific and powerful `path` module. (But this is a test and certainly not subject to that kind of diligence.)

Reply:
I lost track of whether I was in the land of the rich node stdlib or the xs dessert... I guess what I ended up with is a bit shizophrenic.